components/bank/BankBad.vue

```diff
@@ -52,6 +52,19 @@
         :field="bankPage?.data.description3"
       />
     </template>
+     <template #section3>
+      <PrismicRichText
+        v-if="
+          bankPage?.data?.description4 && !isEmptyPrismicField(bankPage?.data.description4)
+        "
+        class="text-center text-blue-900 text-lg md:text-2xl tracking-wider font-semibold mb-4"
+        :field="bankPage.data.description4"
+      />
+      <PrismicRichText
+        class="text-blue-900 leading-loose text-lg"
+        :field="bankPage.data.description5"
```

Comment:
we don't need two new fields but just one. So `description4` is enough. It's a richtext so the copy from both the `h3` and the `p` tag can be now added to the same richtext. You can just select in Prismic that the h3 should be a Header 3. 

So would be good you remove the `description5` again and just use `description4`

Reply:
@sopoour , I removed `description5`, and I tried making heading3 in prismic but it applies to complete `description` rich text body, so just applied paragraph tag to it.  I have pushed my changes. Could you please review and approve if everything looks good?